app/assets/stylesheets/base/_variables.scss

```diff
@@ -13,7 +13,8 @@ $heading-line-height: 1.2;
 $base-border-radius: 4px;
 $base-spacing: $base-line-height * 1rem;
 $small-spacing: $base-spacing / 2;
-$medium-spacing: ($small-spacing * 1.5);
+$medium-spacing: $small-spacing * 1.5;
```

Comment:
Nice move on the incremental spacing! Looks great visually and it'll be easier later on.